### ISSUE TITLE
Adjust how screen readers handle Upcoming Meetings section on homepage

### ISSUE THREAD

**Issue by antidipyramid** [Metro-Records/la-metro-councilmatic#1275]
The Metro team received some feedback that the Upcoming Meetings section of the homepage doesn't work well with screen readers. We should make sure that readers read out meetings sequentially (with meeting name first) and with sufficient demarcation between them.

<img width="867" height="555" alt="Image" src="https://github.com/user-attachments/assets/49f473b0-03c7-44be-b278-d502c9f0132c" />

**Comment by neilarellano** [comment]
These changes are great! I especially love that it reads aloud that it is about to read a new meeting. This is much easier to understand via screenreader.
